src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java

```diff
@@ -474,7 +474,7 @@ private List<File> getCompileClasspathElements(MavenProject project) {
         return list;
     }
 
-    private Map<String, File> getModulePathElements() throws MojoFailureException {
+    Map<String, File> getModulePathElements() throws MojoFailureException {
```

Comment:
Normally, it is not a good practice to change the visibility of a method for testing. 

Do you have another idea to test it? Maybe an integration test instead of a unit test.

You should at least add a comment to the code that you change the visbility for testing.